src/styles/index.css

```diff
@@ -7,7 +7,7 @@ body {
 }
 
 .gradient {
-  background-image: linear-gradient(-225deg, #a5b69f 0%, #51b63c 100%);
+  background-image: linear-gradient(-225deg, #a5b69f 0%, theme('colors.tsfresh-green.500') 100%);
```

Comment:
References to the color should be preferred instead of repeating the color explicitly. See also [here](https://tailwindcss.com/docs/customizing-colors#nested-object-syntax).